sdk/va-report-components/documentation/docs/api/SASReportPageElement.md

```diff
@@ -58,6 +58,15 @@ When `true`, report objects that support zooming require a modifier key be used
 
 default value: `true`
 
+### `reportContextKey: string`
+
+By default SASReportPageElements that have the same `reportUri` or `packageUri` do not share a report context. However, creating a separate report context per page can have negative performance characteristics and the default behavior results in `getReportHandle()` returning a different handle per SASReportPageElement. Setting the same `reportContextKey` value on multiple SASReportPageElements will improve performance in some cases and will result in `getReportHandle()` returning the same instance.
+
+Note: A future release of va-report-components will change the default behavior. In the future SASReportPageElements will share report contexts with other SASReportPageElements and SASReportElements.
+
+default value: `undefined`
+default behavior: unique reportContextKey per SASReportPageElement
+
```

Comment:
```suggestion
### `reportContextKey: string`

`reportContextKey` controls the sharing of report contexts between different `SASReportObjectElement` and `SASReportPageElement` elements that originate from the same report.  A shared report context allows for report object actions like filtering and brushing to occur between elements.  Elements sharing a report context also have the same shared instance of a `ReportHandle`, thus all report parameters are also shared. In contrast, unique report contexts do not allow for interactions across elements and result in a unique `Report Handle`. Unique report contexts also allow for multiple instances of the same report object to be shown at one time, which is not possible when using a shared report context.  By default `SASReportPageElement`s use their own unique report context. Setting identical `reportContextKey` values on elements from the same report will result in use of a shared report context. Doing this can give performance benefits when multiple `SASReportPageElement`s from the same report are used.

default value: `undefined`<br>
default behavior: use a unique report context per `SASReportPageElment`

<span style='color:red'>DEPRECATED</span>: This default behavior is deprecated and will change in a future release of va-report-components. The new default behavior will share report contexts with other `SASReportPageElement` and `SASReportObjectElement` elements from the same report.

```

Reply:
I do like actually calling this behavior change out as a deprecation, in bold red color.